.github/workflows/ci.yml

```
@@ -22,39 +22,33 @@ jobs:
           - { java-version: "11", os: "macos-latest", os-label: "macOS" }
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Set up JDK ${{ matrix.java-version }}
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v4
```

actionlint (1.7.4)

missing input "distribution" which is required by action "actions/setup-java@v4". all required inputs are "distribution"

(action)